pages/affinity-diagramming.md

```diff
@@ -11,7 +11,7 @@ A way of finding themes in collections of ideas, quotes, or observations.
 
 ## Reasons to use it
 
-Quick and collaborative approach for drawing insights from qualitative data.
+To draw out insights from qualitative data quickly and collaboratively.
```

Comment:
Not referencing an issue, just something I noticed the other day. I applied
the `copy edit` label to this PR so it *should* make it to print if the
contrib update is accurate about how we do that.

On Wed, Sep 16, 2015 at 10:51 AM, Colin MacArthur <notifications@github.com>
wrote:

> In pages/affinity-diagramming.md
> <https://github.com/18F/methods/pull/48#discussion_r39639434>:
>
> > @@ -11,7 +11,7 @@ A way of finding themes in collections of ideas, quotes, or observations.
> >
> >  ## Reasons to use it
> >
> > -Quick and collaborative approach for drawing insights from qualitative data.
> > +To draw out insights from qualitative data quickly and collaboratively.
>
> Do we need to put this in a separate PR so it makes it into the printed
> version? Is it referencing an issue? (Good catch, though!)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/18F/methods/pull/48/files#r39639434>.
>